src/storage/src/table/storage_table.rs

```diff
@@ -390,6 +402,11 @@ impl<S: StateStore, E: Encoding> StorageTableBase<S, E, READ_WRITE> {
         buffer: BTreeMap<Vec<u8>, RowOp>,
         epoch: u64,
     ) -> StorageResult<()> {
+        // The epoch might not be generated by meta, but we can use this epoch to filter out keys
+        // from the current epoch. Conflicts in the current epoch should be checked by conflict
+        // detector.
+        let sanity_check_epoch = epoch.checked_sub(1);
```

Comment:
Can we use the current epoch to check? With write-anytime, we should also take the records with the current epoch into consideration.